crates/storage/db/src/implementation/mdbx/cursor.rs

```diff
@@ -218,8 +218,7 @@ impl<K: TransactionKind, T: DupSort> DbDupCursorRO<T> for Cursor<K, T> {
                         .map_err(|e| DatabaseError::Read(e.into()))?
                         .map(|val| decoder::<T>((Cow::Owned(key), val)))
                 } else {
-                    let err_code = MDBXError::to_err_code(&MDBXError::NotFound);
-                    Some(Err(DatabaseError::Read(err_code)))
+                    Some(Err(DatabaseError::Read(MDBXError::NotFound.into())))
```

Comment:
why can we not just store mbdxerror enum instead of converting it into a string?

Reply:
ah mb youre righ